emscripten.py

```diff
@@ -2537,9 +2537,10 @@ def create_sending_wasm(invoke_funcs, forwarded_json, metadata):
   if shared.Settings.SAFE_HEAP:
     basic_funcs += ['segfault', 'alignfault']
 
-  em_asm_sigs = [sigs for _, sigs, _ in metadata['asmConsts'].values()]
+  em_asm_sigs = [zip(sigs, call_types) for _, sigs, call_types in metadata['asmConsts'].values()]
```

Comment:
is the call_types part of this tuple actually used?

Reply:
Yeah, on the other changed line
This change turns em_asm_sigs from a list of signatures into a list of signature+call_type pairs, and we use both parts of the pair to generate the correct name

Reply:
oh, right... i was just confused by the flattening step in between.